cn/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -70,7 +70,7 @@ Kubernetes 满足了生产中运行应用程序的许多常见的需求，例如
 * [Pod](/docs/user-guide/pods/) 提供复合应用并保留一个应用一个容器的容器模型,
 * [挂载外部存储](/docs/user-guide/volumes/),
```

Comment:
I get a 404 here.

Reply:
fixed

Reply:
actually this one is a redirection as well. so I'm not submitting another commit.

Reply:
cool. where is this link redirected to ?

Reply:
https://github.com/kubernetes/website/blob/master/_redirects#L390